benchmarks/CMakeLists.txt

```diff
@@ -66,4 +66,6 @@ endfunction()
 
 if(SYCL_INTEL_TARGET)
   add_subdirectory(pvc)
+else(SYCL_NVIDIA_TARGET)
```

Comment:
```suggestion
if (SYCL_NVIDIA_TARGET)
```

Reply:
In theory there is no limitation to disable/enable one or the othere, each of them have the specific translation unit and if both hardware available, it should be possible to compile both given the dpcpp compiler we are using supports both